internal/github/profile_test.go

```diff
@@ -0,0 +1,282 @@
+package github_test
+
+import (
+	"context"
+	_ "embed"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/chinmina/chinmina-bridge/internal/config"
+	"github.com/chinmina/chinmina-bridge/internal/github"
+	api "github.com/google/go-github/v61/github"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed profile/valid_profile.yaml
+var profile string
+
+//go:embed profile/invalid_profile.yaml
+var invalidProfile string
+
+// Test that the
+func TestURLDecomposition(t *testing.T) {
+	// Example of a valid profile URL
+	configURL := "github.com/chinmina/chinmina-bridge/docs/profile.yaml"
+
+	// Test that the profile URL is valid
+	owner, repo, path := github.DecomposePath(configURL)
+	assert.Equal(t, "chinmina", owner)
+	assert.Equal(t, "chinmina-bridge", repo)
+	assert.Equal(t, "docs/profile.yaml", path)
+
+	// Example of a valid profile URL
```

Comment:
Was this comment meant to be:

```suggestion
	// Example of a invalid profile URL
```
?